src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -182,5 +209,37 @@ private bool IsAttributeDerivedFromClassInternal(ITypeSymbol attributeType, ITyp
 
 		private bool IsAggregatorAttribute(ITypeSymbol attributeType) =>
 			attributeType.InheritsFromOrEquals(_aggregateAttribute) || attributeType.InheritsFromOrEquals(_dynamicAggregateAttribute);
+
+
+		///TODO: refactoring arguments -> remove semanticModel to constructor? Is it nessesary?
+		///TODO: Add DataFlow analyze to corner cases with defaul IsDBBound assigment.
+		public bool? IsBoundField(PropertyDeclarationSyntax property, SemanticModel semanticModel)
```

Comment:
@saloponov @dropsonic I don't see processing of PXDBScalar and PXDBCalced cases.